src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -194,6 +199,22 @@ private static void GetTextResourceForLanguage(Dictionary<string, Dictionary<str
             }
         }
 
+        /// <summary>
+        /// Returns content in text file written in the new text format
+        /// identified by languageCode in filename as a string.
+        /// </summary>
+        /// <param name="languageCode">Language identifier</param>
+        public async Task<string> GetTextContentV2(string languageCode)
+        {
+            string fileName = "text." + languageCode + ".json";
```

Comment:
No biggie, but in general prefer string interpolation over concatenating texts for easier reading and performance (not relevant in this case).